app/DummyControllerApi/Models/EgressBucketResponseModel.cs

```diff
@@ -2,9 +2,27 @@ namespace DummyControllerApi.Models;
 
 public class EgressBucketResponseModel
 {
+  /// <summary>
+  /// Submission ID
+  /// </summary>
+  public string SubId { get; set; } = string.Empty;
+  
+  /// <summary>
+  /// The Minio Host details
+  /// </summary>
   public string Host { get; set; } = string.Empty;
+  
+  /// <summary>
+  /// The name of the target Minio bucket
+  /// </summary>
   public string Bucket { get; set; } = string.Empty;
   
+  /// <summary>
+  /// A path to prefix to object id's uploaded for this submission
```

Comment:
English

Reply:
yes, it is? I don't really know how to say it clearer.

when a submission requests egress bucket details (and therefore gets this response model), this property provides a path value for them to put on the front (prefix) of any object id's for files uploaded as part of the egress request...

but that's what it says in fewer words...

Reply:
I think I got it now. When I read it first, it didn't seem to make sense. Need more caffeine, clearly lol